### DISCUSSION TITLE
有没有办法多 table, 同一份 columns 可以宽度自动对齐?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#45654] [Q&A]
问题: 希望某些列自动撑开, 比如纯数字列, 那么其他 table,也自动调整宽度, 你们考虑支持吗?

**Comment by zombieJ** [comment]
这个直接设置单元格的 word break css 即可，让数字列不允许换行。其他的可以换行或者省略。

**Reply by vaynevayne** [reply]
关键词: 多 table
